src/main/java/seedu/address/model/common/Date.java

```diff
@@ -7,7 +7,7 @@
 import java.time.format.DateTimeFormatter;
 
 public class Date {
-    public static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("yyyy-mm-dd");
+    public static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd");
```

Comment:
Maybe consider YYYY-MM-DD (all upper, instead of partially upper), then hh:mm (all lower)

Reply:
this format follows java syntax, I cant recall the difference hmmm